builtin/config.c

```diff
@@ -189,11 +191,60 @@ static void show_config_origin(struct strbuf *buf)
 	strbuf_addch(buf, term);
 }
 
+static const char *scope_to_string(enum config_scope scope) {
+	const char *local = "(local)";
+	const char *global = "(global)";
+	const char *system = "(system)";
+	const char *command_line = "(command line)";
+	const char *unknown = "(unknown)";
+	/* --local, --global, and --system work the same as --file so there's
+	 * no easy way for the parser to tell the difference when it is
+	 * setting the scope, so we use our information about which options
+	 * were passed
+	 */
+	if (use_local_config) {
```

Comment:
Wouldn't it be more succinct to say

```c
if (use_local_config || scope == CONFIG_SCOPE_REPO)
    return N_("local");
```

?

Reply:
Yeah, that's my gaff